packages/expo-optimize/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "resolveJsonModule": true,
     "rootDir": "src",
     "declaration": false,
-    "composite": false
+    "composite": false,
+    "typeRoots": ["../../ts-declarations", "node_modules/@types"],
```

Comment:
I don't remember how it works but when you specify `node_modules/@types`, does it refer to `./node_modules/@types` or to `ROOT/**/node_modules/@types`? I don't see any `@types/*` libs in the package.json for `expo-optimize`. Maybe it's not necessary then?

Reply:
- If it's only for consistency with other libs then I'm fine with this.
- I'd also want to know what happens if a library is shared between workspaces, i.e. `expo-optimize` and `image-utils` both use `@types/abc123`. Does it end up in the root `node_modules` and if so, does this configuration work correctly in the case?

Reply:
Workspaces _should_ take care of this. Like I mentioned in my notes, I haven't been able to figure out why it doesn't just pick up the `typeRoots` from the base tsconfig in the root of the workspace, so that's what I'm working around here. This is one thing I'd rather fix properly.